### PR TITLE
Jishnu/ansika version managements

### PR DESCRIPTION
**Pull request by jishnupn04**
## Description
1. Added Installation scripts for GVM,NVM,PyEnv,McFly,Rupa/z.

### CODE REVIEW

File: .github/workflows/build-and-release.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ name: Binary Build And Release
 run-name: ${{ github.actor }} building the binary using nuitka 🚀
 on:
   push:
+    branches:
+      - jishnu/AnsikaVersionManagements
```
Comment (LinceMathew) [inline .github/workflows/build-and-release.yml:6]: revert this lines

File: one_installer.yml [hunk 2]
```diff
@@ -9,88 +8,208 @@
   tasks:
     - name: Update package repository
       shell: sudo apt update
+      ignore_errors: yes
```
Comment (LinceMathew) [inline one_installer.yml:11]: what is this rule doing?

Comment (jishnupn04) [reply one_installer.yml:11]: On updating packages,errors come up if there are deprecated dependencies.Then it won't go to the next task.So what ignore_errors does is even if an error comes up,t won't stop the installation of the scripts which comes after this task

File: .github/workflows/build-and-release.yml (hunk 1 above)
Comment (jishnupn04) [reply .github/workflows/build-and-release.yml:6]: Ok

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,10 @@
 .gitlab-ci.yml
 ext.yml
-requirement.txt
\ No newline at end of file
+requirement.txt
```
[the remark this answers is not in the document]
Comment (jishnupn04) [reply .gitignore:3]: OK

**Pull request merged**